### ISSUE TITLE
PyFFTW on Windows needs to be updated

### ISSUE THREAD

**Issue by psavery** [OpenChemistry/tomviz-superbuild#287]
Our PyFFTW on Windows was built with python 3.6. We need to re-build it with python 3.7 so it will work. And, while we are doing it, it might be worth upgrading to pyFFTW 0.11.1 from our previous version, pyFFTW 0.10.4.

In theory, this should be easy, but when I follow the [build instructions](https://github.com/pyFFTW/pyFFTW/blob/v0.11.1/README.rst), I run into a couple of errors I'm working to resolve. I get:
```
[WinError 6] The handle is invalid
```

followed by the full build command and then:
```
error: Could not find the FFTW header 'fftw3.h'
```

I'm going to try out a few more things to make sure the header is in the include path.

I am using the python 3.7 that is on kerbin in the tomviz build at `C:\d\buildbot\fff8b192\build\install\bin\python.exe` to build it. This should hopefully have all of the required dependencies including cython. I am also using the fftw libraries on kerbin at `C:\d\buildbot\fff8b192\build\install\bin\libfftw*.dll`, and the header at `C:\d\buildbot\fff8b192\build\install\include\fftw3.h`.

**Comment by psavery** [comment]
If I try the same steps with `pyFFTW-0.10.4` instead, I don't get that error. It starts to compile, but I run into a different error:
```
error C2039: 'exc_type': is not a member of '_ts'
error C2039: 'exc_value': is not a member of '_ts'
```

I guess we should make sure there are some clear build instructions for pyfftw for Windows when we get it working...

**Comment by psavery** [comment]
So I figured out the problems. There were actually two:

1. I thought our Windows superbuild automatically builds cython, but I guess it doesn't (mac and Linux do). So I needed to build and install cython before attempting to compile pyfftw. That fixes the issue for `pyFFTW-0.10.4`.
2. For `pyFFTW-0.11.1`, it includes an environment sniffer, which checks to make sure that all the files can be found in python before compiling them. This environment sniffer does not seem to work for me on Windows, though. It says that the files are missing when they are actually there. 
Thus, I needed to manually comment the checks out ([here](https://github.com/pyFFTW/pyFFTW/blob/09ecbe3f864eb829241eebf990d4332c4164bb1b/setup.py#L217) and [here](https://github.com/pyFFTW/pyFFTW/blob/09ecbe3f864eb829241eebf990d4332c4164bb1b/setup.py#L323)). If I remove those checks, it compiles fine.

**Comment by psavery** [comment]
This gets fixed by PR #288, but I should mention that it is not upgrading to 0.11.1, but keeping it at 0.10.4. We probably need to update some python code to get 0.11.1 working. I get the following error if I try to run the Direct Fourier Method with 0.11.1 on Windows:
```
Traceback (most recent call last):
  File "Reconstruct (Direct Fourier)", line 1, in <module>
  File "C:/Program Files/tomviz\bin/Lib/site-packages\pyfftw\__init__.py", line 42, in <module>
    from . import builders
  File "C:/Program Files/tomviz\bin/Lib/site-packages\pyfftw\builders\__init__.py", line 3, in <module>
    from .builders import *
  File "C:/Program Files/tomviz\bin/Lib/site-packages\pyfftw\builders\builders.py", line 266, in <module>
    from ._utils import (_precook_1d_args, _Xfftn, _norm_args, _default_effort,
  File "C:/Program Files/tomviz\bin/Lib/site-packages\pyfftw\builders\_utils.py", line 95, in <module>
    raise NotImplementedError("No default precision available")
NotImplementedError: No default precision available
"Failed to create module."
Failed to create module.
```
